src/NEO/CoinReference.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <vector>
+
+#include "ISerializable.h"
+#include "../uint256.h"
+#include "../UInt.hpp"
+#include "../Data.h"
+#include "../ReadData.h"
+
+namespace TW::NEO {
+    class CoinReference : public ISerializable {
```

Comment:
same here, can you remove the indentation?, please also add license header to each file

Reply:
Sure, I'm going to add the license to all files.